mytransactions.sql

```diff
@@ -0,0 +1,30 @@
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+ROLLBACK;
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Verify%20that%20change%20was%20made.%20Then%20roll%20back%20the%20change%20and%20verify%20that%20the%20species%20columns%20went%20back%20to%20the%20state%20before%20the%20transaction.) Please verify that change was made and after the rollback verify that the species columns went back to the state before the transaction.

Reply:
Thank you @Omar-Muhamad, this was verified and captured in the screenshot of the first transaction.